src/current/v24.1/physical-cluster-replication-overview.md

```diff
@@ -9,6 +9,8 @@ CockroachDB **physical cluster replication (PCR)** continuously sends all data a
 
 In a disaster recovery scenario, you can [_cut over_]({% link {{ page.version.version }}/cutover-replication.md %}) from the unavailable primary cluster to the standby cluster. This will stop the replication stream, reset the standby cluster to a point in time where all ingested data is consistent, and mark the standby as ready to accept application traffic.
 
+Physical cluster replication is supported on CockroachDB {{ site.data.products.core }} in new clusters on v23.2 or above.
```

Comment:
hrm could we say v24.1 instead :D?

It's only GA in 24.1 and was public preview in 23.2.

Reply:
I have replaced this sentence with a link to the prerequisites for PCR. Since the PCR docs section is located in the self-deployment section and the docs are versioned, I think the main thing we need to communicate is that users need to be careful of the cluster version. So, that linked section has the necessary details as well as mentioning the non-ua to ua workflow. Let me know if you don't think that works! 
On this overview page I also moved the "Cluster versions and upgrades" section up a little as it was the last part of the page before.

Reply:
Sg. Not a strong opinion.